tldw_Server_API/tests/DB_Management/test_media_postgres_migrations.py

```
@@ -371,3 +374,373 @@ def test_media_postgres_migration_reaches_v22_and_restores_email_schema(
+                    "job-unique",
+                    str(uuid.uuid4()),
+                    "2024-01-06T00:00:00Z",
+                    1,
+                    db.client_id,
+                    False,
+                ),
+                connection=conn,
+            )
+
+        with pytest.raises(BackendDatabaseError, match="unique|duplicate key"):
+            with backend.transaction() as conn:
```

Ruff (0.15.9)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)